gluonnlp/model/train/embedding.py

```diff
@@ -22,20 +22,27 @@
 
 __all__ = ['EmbeddingModel', 'SimpleEmbeddingModel', 'FasttextEmbeddingModel']
 
+import logging
+import struct
+import sys
+
 import numpy as np
 from mxnet import cpu, nd
 from mxnet.gluon import Block, HybridBlock, nn
 
-from ... import embedding as emb
-from ...data.batchify import Pad
+from ...vocab import create_subword_function
+
+if sys.version_info[0] == 3:
```

Comment:
Maybe we want to create a base.py like the one in mxnet for common types like this

Reply:
Good idea